management_api_app/service_bus/service_bus.py

```diff
@@ -0,0 +1,27 @@
+from azure.identity.aio import DefaultAzureCredential
```

Comment:
you should call this service_bus_sender because soon we could have service_bus_receiver

Reply:
Let's see when we get there. There still room for more methods in the class.